CRM/Member/Form/Membership.php

```diff
@@ -121,17 +121,14 @@ class CRM_Member_Form_Membership extends CRM_Member_Form {
    */
   public static function getSelectedMemberships($priceSet, $params) {
     $memTypeSelected = array();
-    $priceFieldIDS = self::getPriceFieldIDs($params, $priceSet);
-    if (isset($params['membership_type_id']) && !empty($params['membership_type_id'][1])) {
```

Comment:
This line was to catch 'non price set' - we need to be sure that still works  (I know we have been working to disolve the distinction & perhaps we already have)

Reply:
ah I see - handled in self::getPriceFieldIDs